lacci/lib/shoes/drawables/para.rb

```diff
@@ -1,7 +1,8 @@
 # frozen_string_literal: true
 
 class Shoes
-  class Para < Shoes::Drawable
+  class Para < Shoes::Drawable 
+    require_relative 'font_helper.rb'
```

Comment:
Minor thing, but it probably makes more sense to put the require up at the top unless it's inside a method. In this case, up top makes more sense.